tests/com/goide/inspections/GoHighlightingTest.java

```diff
@@ -131,7 +132,7 @@ public void testInnerTypesFromOtherPackage() {
 
   public void testDuplicatesNoLocalResolveForTest() {
     myFixture.configureByText("a.go", "package i; type P struct { v1 int }");
-    myFixture.configureByText("b.go", "package i_test; import ( \".\" ); func <warning>f</warning>() { print(i.P{}.<error>v1</error>) }");
+    myFixture.configureByText("b.go", "<error>package i_test</error>; import ( \".\" ); func <warning>f</warning>() { print(i.P{}.<error>v1</error>) }");
```

Comment:
Changing getPackageName() logic broke the initial issue (#1560). With this change you've fixed the test, not the bug. 
I'll fix it, but be careful in the future.